covariancematrix.go

```diff
@@ -25,8 +26,6 @@ func CovarianceMatrix(cov *mat64.Dense, x mat64.Matrix, wts []float64) *mat64.De
 
 	r, c := x.Dims()
 
-	// TODO(jonlawlor): indicate that the resulting matrix is symmetric, which
```

Comment:
OK, I'll put it back.  I just figured that since MulTrans was using the Symmetric route, that this was basically satisfied.  But you're right, when *mat64.Symmetric is available this function should return that type instead.

That brings up a question: do we want CovarianceMatrix to return a mat64.Matrix now, or do we want to change the type to *mat64.Symmetric later?